CMake/cdat_modules_extra/setup_runtime.sh.in

```diff
@@ -10,10 +10,11 @@ function cleanup {
 # which can lead to errors.
 if [ -n "${UVCDAT_SETUP_PATH}" ] ; then
   if [ "${UVCDAT_SETUP_PATH}" = "${install_prefix}" ] ; then
+    echo "Nothing to do UVCDAT is already setup at "${UVCDAT_SETUP_PATH} 1>&2
```

Comment:
actually per: http://wiki.bash-hackers.org/syntax/quoting the quotes are probably not even needed here...